src/main/resources/model/instance.json

```diff
@@ -610,8 +610,7 @@
     "itemEffectiveShelvingOrder": {
       "type": "search",
       "index": "keyword",
-      "searchAliases": [ "callNumber", "items.effectiveShelvingOrder" ],
-      "searchTermProcessor": "effectiveShelvingOrderTermProcessor",
+      "searchAliases": [ "items.effectiveShelvingOrder" ],
```

Comment:
Shouldn't it be 'item.effectiveShelvingOrder' ?

Reply:
Yes you're right. This searchAlias using for backward compatibility.
To have ability to search by "item" we using `ItemSearchFieldModifier` class
From tests you can see that searching by `item.effectiveShelvingOrder` works:
![image](https://user-images.githubusercontent.com/88006020/187649654-d01f0257-662b-47ec-bc73-f1f2c4e862ec.png)

Reply:
Ok, got it, thank you